.github/workflows/build_icons.yml

```diff
@@ -18,13 +17,12 @@ jobs:
           npm install
 
       - name: Executing build and create fonts via icomoon
-        shell: cmd
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: >
           python ./.github/scripts/icomoon_build.py 
-          ./.github/scripts/build_assets/geckodriver-v0.30.0-win64/geckodriver.exe ./icomoon.json 
-          ./devicon.json ./icons ./ %GITHUB_TOKEN% --headless
+          ./.github/scripts/build_assets/geckodriver-v0.32.2-linux64/geckodriver ./icomoon.json 
+          ./devicon.json ./icons ./ "$GITHUB_TOKEN" --headless
```

Comment:
Did this fix the issue with the github token not working? If so, is it because the syntax for referring to a variable is different in bash and cmd?

Reply:
Yeah, it's different. And yes, it's fixed.